src/genezio.ts

```diff
@@ -42,6 +45,17 @@ if (ENABLE_DEBUG_LOGS_BY_DEFAULT) {
     setDebuggingLoggerLogLevel("debug");
 }
 
+// setup package manager
+const configuration = await getProjectConfiguration();
+if (configuration.packageManager) {
+    if (configuration.packageManager in Object.keys(PackageManagerType)) {
+        setPackageManager(configuration.packageManager);
+    } else {
+        log.warn(`Unknown package manager '${configuration.packageManager}'. Using 'npm' instead.`);
+        setPackageManager(PackageManagerType.npm);
+    }
+}
```

Comment:
@vladiulianbogdan I am not sure if this code should stay here or be moved in `src/packageManagers/packageManager.ts`. The latter option has the advantage that all code related to the package manages stays together. What do you think?

Reply:
I think it is better here. If we move this to the `packageManager.ts`, then we pollute a class with yaml and project logic. This file is the one that does the wiring, so to respect the single responsibility principle, I think it is better to have this logic here.